### ISSUE TITLE
Doesn’t pick up on invalid empty identifier/source

### ISSUE THREAD

**Issue by jenstroeger** [w3c/epubcheck#1117]
It would be helpful if EpubCheck would info/warn that identifiers are empty (or don’t match a pattern). For example,
```xml
<dc:identifier id="id-e985e6af">urn:isbn:</dc:identifier>
<meta refines="#id-e985e6af" property="identifier-type" scheme="onix:codelist5">15</meta>
<dc:source id="source-e985e6af">urn:isbn:</dc:source>
<meta refines="#source-e985e6af" property="source-of">pagination</meta>
```
isn’t being flagged at all. Same goes for UUIDs instead of ISBNs. I guess it would be ok with the elements contained _no_ text whatsoever (in which case maybe `<meta>` could be flagged), but a n existing `urn:isbn:` expresses an intent to use an identifier, but there’s none…

**Comment by rdeltour** [comment]
It isn't a spec violation, since EPUB basically allows any string as an identifier, but yes I can see how it can be useful.

There is a precedent of implementing UUID validation in #853 (so I'm surprised you say the same goes for UUIDs?).

It's low priority, but PRs are always welcome if anyone wants to fast-track an implementation 😊

**Comment by jenstroeger** [comment]
Thanks, @rdeltour!

> It's low priority, but PRs are always welcome if anyone wants to fast-track an implementation 😊

If you can point me at the vicinity of where the responsible code lives, I’d take a stab at it.
